corehq/motech/dhis2/tests/test_entities_helpers.py

```diff
@@ -594,12 +728,62 @@ def test_happy_path(self):
 
 
 def test_doctests():
-    from corehq.motech.dhis2 import entities_helpers
-
     results = doctest.testmod(entities_helpers)
     assert results.failed == 0
 
 
+def test_get_tracked_entity_relationship_specs_should_return_a_valid_instance():
+    relationship_type_id = "TDEdu7DAEQL"
```

Comment:
`nit`
I personally would avoid testing this since the only logic it has is `tracked_entity and hasattr(tracked_entity, 'relationships` which should be tested or refactor the method to do the logic and object initialisation seperately, so that only logic can be tested.

Reply:
Good point. Yeah I think we can chuck this test then. The relationship tests will cover this method's return values (`test_no_relationships` covers the empty array return type).

Reply:
👍